examples/multiple-apps.ts

```diff
@@ -7,20 +7,31 @@ import { pause, startAnalysis } from '../src/index';
 // `npx tsx examples/multiple-apps.ts <emulator name> <snapshot name> <path to a folder of single APKs>`
 
 (async () => {
-    const emulatorName = process.argv[2] || 'emulator-name';
-    const snapshotName = process.argv[3] || 'snapshot-with-setup-emu';
-    const apkFolder = process.argv[4] || 'path/to/app-files';
+    const apkFolder = process.argv[2];
+    const emulatorName = process.argv[3];
+    const snapshotName = process.argv[4];
```

Comment:
This seems like it was only for testing?

Reply:
Its also useful so you don’t have to think of any example values.

Reply:
But there were examples already? Anyway, I don't care about the change but if you do change the param order, you also need to change the usage comment.